### DISCUSSION TITLE
Consider adding markdown support for Readwise tag conventions

### DISCUSSION THREAD

**Discussion by flatlines** [hadynz/obsidian-kindle-plugin#142] [Ideas]
Hello,

I love the idea of this plugin. I have been using Readwise for years and one of my favourite features is [chapter tagging](https://blog.readwise.io/add-chapters-to-highlights/) in highlights. Essentially you highlight a chapter heading, add a note (.h1) and the heading is automatically recognised by RW as a chapter heading. They do the same for concatenating paragraphs, where you can use .c1, .c2 etc. to string different snippets together.

I will likely stay a RW user as they support Instapaper and other tools, but the conversion of chapter headings into markdown is an incredible timesaver & I suspect many might benefit.

Best wishes,
Flatlines

**Comment by hadynz** [comment]
This is a brilliant idea, and I think would help everyone significantly be able to scan through their highlights. 

I'll take it on as a feature to add to the plugin.

**Reply by iamthew4lrus789** [reply]
Would love this feature too - it would allow this plugin to supplant Readwise entirely for me. Thank you!

**Reply by hadynz** [reply]
I'm halfway through implementing this feature at the moment. Will be included in the next release.

**Reply by iamthew4lrus789** [reply]
Amazing! Thank you

**Reply by iamthew4lrus789** [reply]
PS. Bought you a coffee through the link on the plugin page - thanks again.
